src/locales/en/messages.json

```diff
@@ -2989,5 +2989,41 @@
   },
   "passwordGeneratorPolicyInEffect": {
     "message": "One or more organization policies are affecting your generator settings."
+  },
+  "masterPasswordPolicyInEffect": {
+    "message": "One or more organization policies require your master password to meet the following requirements:"
+  },
+  "policyInEffectMinComplexity": {
+    "message": "Minimum complexity score of $SCORE$",
+    "placeholders": {
+      "score": {
+        "content": "$1",
+        "example": "4"
```

Comment:
I think we need to change this to Weak, Good, Strong instead of the cryptic number score.

Reply:
Will revisit in another PR